src/popup.tsx

```diff
@@ -60,6 +58,29 @@ const Popup = () => {
       </div>
 
       <div className="flex flex-col gap-y-2 mb-2">
+        <form
+          onSubmit={(e) => {
+            setNewType('');
+            setTypes([...types, newType]);
+            e.preventDefault();
```

Comment:
Nice update. I think the "Save Group Types" button can be deprecated since we use the form to manage the stats now, it's much more intuitive
Could you help add the "save" logic in "add new type" and "delete type"?